python/src/aiconfig/editor/client/src/components/EditorContainer.tsx

```diff
@@ -100,15 +170,31 @@ export default function EditorContainer({
       <Container maw="80rem" className={classes.promptsContainer}>
         {aiconfigState.prompts.map((prompt: any, i: number) => {
           return (
-            <PromptContainer
-              index={i}
-              prompt={prompt}
-              key={prompt.name}
-              onChangePromptInput={onChangePromptInput}
-              onUpdateModelSettings={onUpdatePromptModelSettings}
-              onUpdateParameters={onUpdatePromptParameters}
-              defaultConfigModelName={aiconfigState.metadata.default_model}
-            />
+            <Stack key={prompt.name}>
+              <PromptContainer
+                index={i}
+                prompt={prompt}
+                onChangePromptInput={onChangePromptInput}
+                onUpdateModelSettings={onUpdatePromptModelSettings}
+                onUpdateParameters={onUpdatePromptParameters}
+                defaultConfigModelName={aiconfigState.metadata.default_model}
+              />
+              {i ===
+                aiconfigState.prompts.length -
+                  1 /* TODO: Remove this check once we can add prompts at any index */ && (
```

Comment:
Hasn't this been added now? 
https://github.com/lastmile-ai/aiconfig/pull/599

Reply:
Ah weird. I could have sworn my local copy didn't have it when I cmd+clicked in... But it is there now so it's a quick add. Will do so here

Reply:
https://github.com/lastmile-ai/aiconfig/assets/5060851/46ecfef1-df55-4cf1-80eb-fe55dc1667fc